.gitignore

```diff
@@ -5,3 +5,4 @@
 dist/
 docs/_build/
 __pycache__
+.idea
```

Comment:
I think you should use your global gitignore to exclude files generated by your IDE.